### PR TITLE
Header

### PR DESCRIPTION
**Pull request by sunr1**
Created an introduction header and added some logos for some of my profiles.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,33 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+        <div id = "intro">
+            <p>Hi I'm</p>
+            <div id="name">
```
Comment (kzweller) [inline portfolio/src/main/webapp/index.html]: If the purpose of this div is simply for styling reasons, just set the ID on the h2 itself.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,33 @@
...
+                <h2>Rebecca Sun</h2>
+            </div>
+            <p>CS major at Vanderbilt University</p>
+        </div>
+        <div id="about">
+            <p>About</p>
+        </div>
+      <div id="profiles">
```
Comment (kzweller) [inline portfolio/src/main/webapp/index.html]: Indentation.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,33 @@
...
+        <h3>Let's Connect:</h3>
+        <h4>LinkedIn</h4>
+        <img src="https://img.icons8.com/color/48/000000/linkedin.png"/>
+        <h4>Github</h4>
+        <img src="https://img.icons8.com/ios-filled/50/000000/github.png"/>
```
Comment (kzweller) [inline portfolio/src/main/webapp/index.html]: Make sure to provide "alts" : https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,33 @@
...
+      </div>
+      <footer id="footer">
+          <a href="https://icons8.com/icon/13930/linkedin">LinkedIn icon by Icons8</a>
+          <a href="https://icons8.com/icon/12599/github">GitHub icon by Icons8</a>
```
Comment (kzweller) [inline portfolio/src/main/webapp/index.html]: Out of curiosity, why add these?

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -11,18 +11,3 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
Comment (kzweller) [inline portfolio/src/main/webapp/script.js]: Just delete this file if you're not using it.

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -2,8 +2,37 @@
...
+  text-align: center;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#name {
+    font-family: 'Inconsolata', monospace;
+    font-weight: 200;
+    font-size: 32px;
 }
```
Comment (kzweller) [inline portfolio/src/main/webapp/style.css]: This whole file should be alphabetized with declaration order: 
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Also, shorthand things like "font":

https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

**Review by kzweller — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (tambchop) [inline portfolio/src/main/webapp/index.html]: I assume you're going to come back with a future change to make these two images links to your LinkedIn and GitHub pages?

**Review by tambchop — APPROVED** [review]
Looks good so far!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (sunr1) [reply portfolio/src/main/webapp/index.html]: yep! just wanted to have a general visual for my layout at first

**Review by kzweller — APPROVED** [review]

**Pull request merged**